src/Agent.Worker/ResourceMetricsManager.cs

```diff
@@ -173,8 +171,8 @@ private async Task GetCpuInfoAsync()
                     Trace.Error($"Error on receiving CPU info: {message.Data}");
                 };
 
-                var filePath = "grep";
-                var arguments = "\"cpu \" /proc/stat";
+                var filePath = "/bin/bash";
```

Comment:
Can the `bash` be installed in another dir?

Reply:
Thanks for the comment! Actually bash is missing on alpine images, so current approach is slightly incorrect. I'll rework this.